kubernetes/overlays/agreement-email-sender/dev-refactor/deployment.yaml

```diff
@@ -46,13 +86,13 @@ spec:
             - name: SMTP_USERNAME
               valueFrom:
                 secretKeyRef:
-                  name: smtp-certified
-                  key: PEC_SMTP_USER
+                  name: metrics-reports
```

Comment:
I want to generalize this secret, but in an another PR.

Reply:
ok, may I ask you to open a task then?

Reply:
The metrics-report secret contains credentials for the reports sender address, we need a separate K8s secret for the notification address used for agreements (secrets ready on SecretsManager)

Reply:
[Here](https://github.com/pagopa/pdnd-interop-platform-deployment/pull/538) the solution for this thread